.github/workflows/push.yml

```diff
@@ -0,0 +1,68 @@
+name: GitHub Actions
+on: [push]
+env:
+  APP_IMAGE: keboola-ex-db-informix
+  AWS_ACCESS_KEY_ID: "AKIATDSWT5243C6OWPU7"
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+  DOCKERHUB_USER: ${{ secrets.DOCKERHUB_USER }}
+  KBC_DEVELOPERPORTAL_VENDOR: "keboola"
+  KBC_DEVELOPERPORTAL_APP: "keboola.ex-db-hive"
+  KBC_DEVELOPERPORTAL_USERNAME: "keboola+github_actions_db_extractor_informix"
+  KBC_DEVELOPERPORTAL_PASSWORD: ${{ secrets.KBC_DEVELOPERPORTAL_PASSWORD }}
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Print Docker version
+        run: |
+          docker -v
+      - name: Build image and run tests
+        run: |
+          docker login --username "$DOCKERHUB_USER" --password "$DOCKERHUB_TOKEN"
+          docker build --build-arg AWS_ACCESS_KEY_ID --build-arg AWS_SECRET_ACCESS_KEY -t $APP_IMAGE .
+      - name: Set image tag
+        id: tag
+        run: |
+          TAG="${GITHUB_REF##*/}"
+          IS_SEMANTIC_TAG=$(echo "$TAG" | grep -q '^v\?[0-9]\+\.[0-9]\+\.[0-9]\+$' && echo true || echo false)
+          echo "Tag = '$TAG', is semantic tag = '$IS_SEMANTIC_TAG'"
+          echo "::set-output name=app_image_tag::$TAG"
+          echo "::set-output name=is_semantic_tag::$IS_SEMANTIC_TAG"
+      - name: Push image to ECR
+        uses: keboola/action-push-to-ecr@master
+        with:
+          vendor: ${{ env.KBC_DEVELOPERPORTAL_VENDOR }}
+          app_id: ${{ env.KBC_DEVELOPERPORTAL_APP }}
+          username: ${{ env.KBC_DEVELOPERPORTAL_USERNAME }}
+          password: ${{ env.KBC_DEVELOPERPORTAL_PASSWORD }}
+          tag: ${{ steps.tag.outputs.app_image_tag }}
+          push_latest: ${{ steps.tag.outputs.is_semantic_tag }}
+          source_image: ${{ env.APP_IMAGE}}
+  tests:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
tady by si měl stáhnout zpátky image který si poslal do ecr, tak aby se testovalo oproti zbuildovanýmu image

Reply:
A jak k tomu nahodím tu DB pak?

Reply:
no normálně přes docker-compose... musíš ten image mít stejně pojmenovaný, pak ho nebude buildovat ale použije ten stažený... stejně jak je to třeba u mysql extractoru
https://github.com/keboola/db-extractor-mysql/blob/master/.github/workflows/push.yml

Reply:
jo táák